lib/reverse_sentence.rb

```diff
@@ -1,6 +1,15 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
+# Space complexity: O(1)
+
+# combination of reverse_words and string_reverse alorithms from before
+require 'pry'
+require_relative '../../reverse_words/lib/reverse_words'
+require_relative '../../string_reverse/lib/string_reverse'
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  # first reverse letters in the words ex. "Hello World!" becomes "olleH !dlroW"
+  reverse_words(my_sentence)
```

Comment:
👏This helper method trick was quite clever, I'll give you that. 🙂

(Note that you can't always do this in production scenarios, unless you explicitly specify these as dependencies.)